pySDC/tests/test_projects/test_pintsime/test_estimation_check_2capacitors.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+
+@pytest.mark.base
+def test_main():
+    from pySDC.projects.PinTSimE.estimation_check_2capacitors import check
+
+    check()
```

Comment:
Why do you test both the estimation check and the standard model for 2 capacitors, but for 1 capacitor you test nothing?